src/Process/Runner/MutationTestingRunner.php

```diff
@@ -75,13 +76,11 @@ public function __construct(
      */
     public function run(array $mutations, int $threadCount, string $testFrameworkExtraOptions): void
     {
-        $mutantCount = count($mutations);
```

Comment:
Why is this change needed (oncluding change on 96 line)?

Reply:
It's no longer necessary, leftovers of reworking the previous PR

Reply:
Well actually can be kept: for now there is a 1-1 mapping between the number of mutations and created mutant processes so it avoids to do the count twice. It's a micro-optimization though so not necessary per se

Reply:
it's a minor thing, up to you. I would kept it as it was just to not touch already existing code and not counting twice